samples/cpp/benchmark_app/main.cpp

```diff
@@ -1094,9 +1094,9 @@ int main(int argc, char* argv[]) {
 
         if (!FLAGS_exec_graph_path.empty()) {
             try {
-                std::string fileName = fileNameNoExt(FLAGS_exec_graph_path);
-                ov::pass::Serialize serializer(fileName + ".xml", fileName + ".bin");
-                serializer.run_on_model(std::const_pointer_cast<ov::Model>(compiledModel.get_runtime_model()));
+                ov::serialize(std::const_pointer_cast<ov::Model>(compiledModel.get_runtime_model()),
```

Comment:
Suggesting to update docs/snippets as well:
C++: https://github.com/openvinotoolkit/openvino/blob/3aa525c003e5d9de69eb52fe65c2e071102cccfa/docs/snippets/ov_preprocessing.cpp#L195
Python: https://github.com/openvinotoolkit/openvino/blob/3aa525c003e5d9de69eb52fe65c2e071102cccfa/docs/snippets/ov_preprocessing.py#L214

Reply:
done

Reply:
done